StudioCore/ParamEditor/ParamEditorScreen.cs

```diff
@@ -1049,35 +1062,63 @@ public void ParamView(bool doFocus)
                 }
                 scrollTo = 0;
 
-                List<int> pinnedRowList = new List<int>(_paramEditor._projectSettings.PinnedRows.GetValueOrDefault(activeParam, new List<int>()));
-                foreach (int rowID in pinnedRowList)
+                ImGui.Text("id VALUE | name ROW | prop FIELD VALUE | propref FIELD ROW\n | original | modified");
+                UIHints.AddImGuiHintButton("MassEditHint", ref UIHints.SearchBarHint);
+
+                //Goto ID
+                if (ImGui.Button("Goto ID <Ctrl+G>") || InputTracker.GetControlShortcut(Key.G))
                 {
-                    PARAM.Row row = para[rowID];
-                    if (row == null)
-                    {
-                        _paramEditor._projectSettings.PinnedRows.GetValueOrDefault(activeParam, new List<int>()).Remove(rowID);
-                        continue;
-                    }
-                    RowColumnEntry(activeParam, null, para[rowID], dirtyCache, decorator, ref scrollTo, false, true);
+                    ImGui.OpenPopup("gotoParamRow");
                 }
-
-                if (pinnedRowList.Count != 0)
+                if (ImGui.BeginPopup("gotoParamRow"))
                 {
-                    ImGui.NewLine();//Separator bleeds into other columns because imgui
+                    int gotorow = 0;
+                    ImGui.SetKeyboardFocusHere();
+                    ImGui.InputInt("Goto Row ID", ref gotorow);
+                    if (ImGui.IsItemDeactivated())
+                    {
+                        _gotoParamRow = gotorow;
+                        ImGui.CloseCurrentPopup();
+                    }
+                    ImGui.EndPopup();
                 }
 
-                ImGui.Text("id VALUE | name ROW | prop FIELD VALUE | propref FIELD ROW\n | original | modified");
-                UIHints.AddImGuiHintButton("MassEditHint", ref UIHints.SearchBarHint);
 
+                //Row ID/name search
                 if (InputTracker.GetControlShortcut(Key.F))
                     ImGui.SetKeyboardFocusHere();
+                //ImGui.SetNextItemWidth(ImGui.GetColumnWidth(1) - 256);
```

Comment:
We should probably not introduce new commented out lines like these in the codebase, unless you have a comment explaining what it was for and why you still want to keep it around.